package/MDAnalysis/analysis/bat.py

```diff
@@ -25,7 +25,7 @@
 
 :Author: Soohaeng Yoo Willow and David Minh
 :Year: 2020
-:Copyright: GNU Public License, v2 or any higher version
+:Copyright: Lesser GNU Public License, v2 or any higher version
```

Comment:
```suggestion
:Copyright: Lesser GNU Public License, v2.1 or any higher version
```
?

Reply:
Or
```suggestion
:Copyright: Lesser GNU Public License v2.1+
```
for consistency with other files?

Reply:
I noticed we use both interchangeably, so either is fine by me.

Reply:
Spell out "Copyright: Lesser GNU Public License, v2 or any higher version" in top-level places. The "+" is ambiguous.

However, shouldn't it be
```suggestion
:Copyright: Lesser GNU Public License, v2.1 or any higher version
```

Reply:
@RMeli I basically agree with your change, given that we use "v2.1+" in most other files.

I'll have a quick look about the v2 vs v2.1.

Reply:
I don't think v2 exists. From the license itself:
> [This is the first released version of the Lesser GPL.  It also counts
  as the successor of the GNU Library Public License, version 2, hence
  the version number 2.1.]

Reply:
Yes – so all our license text should be referencing 2.1 to be clear.

Most files contain
```
Released under the Lesser GNU Public Licence, v2.1 or any higher version
```
so that should also be the one for this file (your first suggestion, which I hand't properly read — sorry).

Reply:
> However, shouldn't it be

Yes, my comment was about `v2` vs `v2.1`. The two alternative suggestions are because we use either in different places (I don't think it's important to have full consistency TBH).

Reply:
Ah good catch @RMeli! didn't see that one 😅